### DISCUSSION TITLE
University AI training on Aug 22, 2023

### DISCUSSION THREAD

**Discussion by zhuo-yoyowz** [openvinotoolkit/openvino_notebooks#1259] [Show and tell]
Please leave your results after running OpenVINO Notebooks here. Thanks!

**Comment by PnutCN** [comment]
## 402
Running on the Apple M2 Pro
<img width="885" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/58359492/7db6f398-19c2-49a6-b822-739b0f335444">

**Comment by lksr1201** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/71975350/bd5beb0e-cfee-43b3-a0ff-41f629d6ac5b)

**Comment by MyronXie** [comment]
Hello everyone, I'm the TA of the course. If you have any question, please ask me anytime😊

**Comment by Xian-Gao** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/69408351/44c535c8-c84e-47ab-9606-712ff74a2238)

**Reply by Xian-Gao** [reply]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/69408351/cf036d5a-2436-4d39-a1ef-f42e20cf8bb7)

**Comment by jack121381** [comment]
<img width="481" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/115762916/71a808c0-93ac-420f-b88b-bced09251c41">

**Comment by SheldonDong** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/142861147/919fae39-c077-498a-a20d-483013b79b5c)

**Comment by zxclss** [comment]
<img width="1506" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/56336723/5b2ea968-803f-494a-b73e-401e86f58dc4">

**Comment by kAAAsumiGzz** [comment]
<img width="1250" alt="group--13" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/142800932/ec4f00f3-3d37-41fe-bff0-b85fe41c0b5c">

**Comment by yangzccc666** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/69101423/8b126a50-ec0a-4a11-961a-d56f4914e3ef)

**Comment by SUsususususu998** [comment]
<img width="551" alt="1" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/97903247/5c64de75-4ed6-44f2-8a05-f17f7f5f5f9f">

**Comment by EFI-HU** [comment]
<img width="480" alt="cba4b4718777a86f16521e7fcb3847d" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/128896201/11bb1837-181c-4ef2-802f-d01b988ad250">

**Comment by Yeqiangliao** [comment]
<img width="485" alt="cf1d741b1f7f17bf327bb6a0bb8efac" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/142862731/c3337b30-d68c-404e-bc16-27c1cb1c4b5d">

**Comment by deng643793546** [comment]
<img width="1239" alt="e89a36090c6723e9b2eca2d6a1408da" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/138109215/6cc80265-5060-479d-91ea-84a4bc71fee6">

**Comment by susu123321** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/52404165/f9f02e1c-e6fd-41ff-b59c-426a39011698)

**Comment by XingchengZhu** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/100496797/868230c8-90f5-4f0b-a189-864cdd573fdf)

**Comment by KevinW617** [comment]
<img width="548" alt="34bd096bcf50c519993a7fa1a0b7754" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/139970056/566869e4-b999-4332-8e77-528d73ffa058">

**Comment by GGgary666** [comment]
![图片1](https://github.com/openvinotoolkit/openvino_notebooks/assets/56987862/746ff07d-649d-44b2-ac1d-ef57c877c778)

**Comment by jiahang-q** [comment]
![ffc8e5b75b00c6d14623763b9bae0fb](https://github.com/openvinotoolkit/openvino_notebooks/assets/142865640/e7359513-6eac-47b6-ad43-86a83a4c9876)

**Comment by sun-ying-ying** [comment]
<img width="565" alt="屏幕截图 2023-08-22 122721" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/139131851/48947e2f-7534-41e1-aea2-86fc18ac4d99">

**Comment by joki-sr** [comment]
![inspect](https://github.com/openvinotoolkit/openvino_notebooks/assets/115058227/9a8f4180-ecc5-493f-887c-fa30849a6418)

**Comment by White347935991** [comment]
![85eb40247cdb7035b22d9ee598e0c4f](https://github.com/openvinotoolkit/openvino_notebooks/assets/94439166/3be75547-4ddd-48b2-8df5-60ed57b74b3d)

**Comment by xiaolaji986** [comment]
![b22dc0b1d736b1b3faf486efb96444c](https://github.com/openvinotoolkit/openvino_notebooks/assets/105088783/a1404d33-e9ee-4922-b39a-105c1fb47813)

**Comment by zhwgtfr** [comment]
<img width="481" alt="wo" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/142867677/ff670b34-887b-4571-a71a-40270d3696ea">

**Comment by Heimdall-Nss** [comment]
<img width="1205" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/58160555/58c03bbb-b530-4ce8-8856-332531e9c58a">

**Comment by Luciferrrrrrrrrrrrrr** [comment]
### 403
![作业照片1](https://github.com/openvinotoolkit/openvino_notebooks/assets/96982267/40e0fef8-04b8-45ff-af55-00435f43bd45)

**Comment by ChenGeno** [comment]
![171ac8c1d0ffeaed75977350f18a42f](https://github.com/openvinotoolkit/openvino_notebooks/assets/41165687/0c7e145d-fad0-414d-95fe-207c5e666239)

**Comment by XingchengZhu** [comment]
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/100496797/fbf43bba-cc9d-42c3-bef2-d262612630f6)
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/100496797/9399e412-df87-4e99-8b3a-4d4c7f79e6fc)
